portfolio/src/main/webapp/script.js

```diff
@@ -93,14 +93,37 @@ function drawChart() {
   ]);
   const options = {
     'title': 'Zoo Animals',
+    'pieHole': 0.4,
     'width':500,
     'height':400
   };
-  const chart = new google.visualization.PieChart(document.getElementById('chart-container'));
+  const chart = new google.visualization.PieChart(document.getElementById('chart-div'));
   chart.draw(data, options);
 }
 
+function drawStockChart() {
+  var data = new google.visualization.DataTable();
+  data.addColumn('number', 'Week');
+  data.addColumn('number', 'Stock A');
+  data.addColumn('number', 'Stock B');
+  data.addRows([
+    [1,  50, 70],
+    [2,  130, 100],
+  ]);
+  var options = {
+    chart: {
+      title: 'Weekly Performcance of Stock A and Stock B',
+      subtitle: 'in dollars (USD)'
+    },
+    width: 900,
+    height: 500
+  };
+  var chart = new google.charts.Line(document.getElementById('linechart_material'));
+  chart.draw(data, google.charts.Line.convertOptions(options));
```

Comment:
Let's use `const`s instead of `var`s where possible.

Reply:
Addressed in "remove vars"